package.json

```diff
@@ -40,9 +40,11 @@
     "@types/node": "^13.11.1",
     "@types/react": "^16.9.33",
     "@types/react-dom": "^16.9.6",
+    "analytics-client": "^0.5.3",
     "arch": "^2.1.1",
     "balena-sdk": "^13.0.0",
     "lodash-es": "^4.17.15",
+    "mixpanel-browser": "^2.35.0",
```

Comment:
This is in devdependencies in analytics-client but it fails to start if not added, so we need to move it to dependencies.

Reply:
Will fix, thanks.